tickerview/src/main/java/com/vinay/ticker/lib/TickerView.java

```diff
@@ -37,7 +37,9 @@ public class TickerView extends HorizontalScrollView {
     private TextView lastTicker;
     private List<View> childViews = null;
     private LinearLayout linearLayout;
-
+    private Integer replays;
+    private Integer loop = 1;
```

Comment:
Use primitive type for better memory usage?
```suggestion
    private int replays;
    private int loop = 1;
```